### PR TITLE
Block queries to Phenomizer when cases don't have any HPO term assigned

### LINKED ISSUE #1772: Requesting Phenomizer on case with no HPO terms causes server error
```
2020-03-03 16:16:03,274 - scout.server.app - ERROR: Exception on /cust000/4625877fam/phenotypes/actions [POST] [in /home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
 File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
   response = self.full_dispatch_request()
 File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
   rv = self.handle_user_exception(e)
 File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
   reraise(exc_type, exc_value, tb)
 File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
   raise value
 File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
   rv = self.dispatch_request()
 File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
   return self.view_functions[rule.endpoint](**req.view_args)
 File "/home/hiseq.clinical/SERVER/apps/scout/git/scout/scout/server/blueprints/cases/views.py", line 810, in phenotypes_actions
   diseases = controllers.hpo_diseases(username, password, hpo_ids)
 File "/home/hiseq.clinical/SERVER/apps/scout/git/scout/scout/server/blueprints/cases/controllers.py", line 609, in hpo_diseases
   result for result in results if result["p_value"] <= p_value_treshold
 File "/home/hiseq.clinical/SERVER/apps/scout/git/scout/scout/server/blueprints/cases/controllers.py", line 609, in <listcomp>
   result for result in results if result["p_value"] <= p_value_treshold
 File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/query_phenomizer/utils.py", line 120, in query
   raw_result = query_phenomizer(usr, pwd, *hpo_terms)
 File "/home/hiseq.clinical/miniconda/envs/prod/lib/python3.6/site-packages/query_phenomizer/utils.py", line 100, in query_phenomizer
   raise RuntimeError("Phenomizer returned a bad status code: %s" % r.status_code)
RuntimeError: Phenomizer returned a bad status code: 500
```

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #1772

**How to test**:
1. Current master's case page crashes when a query to Phenomizer is launched with empty HPO terms list (no HPO terms assigned)
1. Install this branch on stage and the Phenomizer button will be inactive whenever the case has no phenotypes assigned (so users won't be able to make the interface crash!)

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

File: scout/server/blueprints/cases/templates/cases/phenotype.html [hunk 1]
```diff
@@ -218,7 +218,8 @@
           </div>
           {% if config.PHENOMIZER_USERNAME %}
             <div class="form-group mb-2 mx-sm-3 col-3">
-              <button class="btn btn-outline-secondary btn-sm form-control" type="submit" name="action" value="PHENOMIZER">Phenomizer</button>
+              <button class="btn btn-secondary btn-sm form-control" type="submit" name="action" value="PHENOMIZER"
+              {%if case.phenotype_terms|length == 0 %} disabled {%endif%}>Phenomizer</button>
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/phenotype.html]: :+1:

**Comment by northwestwitch** [comment]
> Just remember the workflows?

What workflows?

**Comment by dnil** [comment]
GitHub actions. I think it is fine now, isn’t it?

**Comment by northwestwitch** [comment]
> GitHub actions. I think it is fine now, isn’t it?

Ahhh, it's fine because it's gone, once we merged the clinsig one, where I had removed the actions!

**Comment by northwestwitch** [comment]
Merging this one as well!

**Pull request merged**